src/CONST.js

```diff
@@ -2526,6 +2526,10 @@ const CONST = {
     TRANSLATION_KEYS: {
         ATTACHMENT: 'common.attachment',
     },
+    TEACHER_UNITE: {
+        PUBLIC_ROOM_ID: '795574440662771',
+        SCHOOL_PRINCIPAL_POLICY_ID: '0A4315E7509A88E0',
```

Comment:
I think we can just use `POLICY_ID` here. The policy is not tied to the principal.

```suggestion
        POLICY_ID: '0A4315E7509A88E0',
```